pkg/manifests/manifests.go

```diff
@@ -3076,8 +3076,9 @@ func (f *Factory) TelemeterClientDeployment(proxyCABundleCM *v1.ConfigMap, s *v1
 
 	// Set annotation on deployment to trigger redeployments
 	if s != nil {
-		hash := sha256.New()
-		d.Spec.Template.Annotations["telemeter-token-hash"] = string(hash.Sum(s.Data["token"]))
+		h := fnv.New64()
```

Comment:
Can we have some quick note about the motivation for the change betweeh sha256 and the fnv hash? Thanks!

Reply:
added a description in the commit message.